app/uk/gov/hmrc/plasticpackagingtax/registration/views/phone_number_page.scala.html

```diff
@@ -54,7 +54,7 @@
             errorMessage = phoneNumberField.error.map(err => ErrorMessage(content = Text(messages(err.message)))),
             label = Label(
                 isPageHeading = true,
-                classes = s"govuk-label--xl",
```

Comment:
I haven't realised that we are moving from `XL` to `L`. Fair enough.
We may reverse this. but that is a Dave/Andy thing.

Reply:
yeah, everywhere else we've got L, easy enough to change if they decide otherwise